tests/end-to-end/director/runtime_api_test.go

```diff
@@ -432,6 +432,64 @@ func TestQuerySpecificRuntime(t *testing.T) {
 	assert.Equal(t, createdRuntime.Name, queriedRuntime.Name)
 }
 
+func TestQueryRuntimesWithPagination(t *testing.T) {
+	//GIVEN
+	ctx := context.Background()
+	runtimeNameSuffix := "runtime"
+	runtimesAmount := 10
+	for i := 0; i < runtimesAmount; i++ {
+		runtimeInput := graphql.RuntimeInput{
+			Name: fmt.Sprintf("%s-%d", runtimeNameSuffix, i),
+		}
+		runtimeInputGQL, err := tc.graphqlizer.RuntimeInputToGQL(runtimeInput)
+		require.NoError(t, err)
+
+		createReq := fixCreateRuntimeRequst(runtimeInputGQL)
+
+		runtime := graphql.Runtime{}
+		err = tc.RunQuery(ctx, createReq, &runtime)
```

Comment:
Please do it in one request

Reply:
We always use loop for creating several resources on server. For me is clean. If you prefer I can change it.

Reply:
It would be faster to do everything in single HTTP request, right?

Reply:
Leave it, but please create a follow-up task for that.

Reply:
Created #193